src/screens/authentication/components/Auth/RecoveryEmail/style.js

```diff
@@ -0,0 +1,23 @@
+import { StyleSheet } from 'react-native'
+import { scale, verticalScale } from 'react-native-size-matters'
+import { theme } from '~/constants'
```

Comment:
Add a blank line after imports.

In general, add blank lines to separate between logical groupings, which is a group of instructions grouped by functionality, semantic purpose or goal.